src/status_im/contexts/wallet/send/input_amount/view.cljs

```diff
@@ -120,6 +121,20 @@
      :title           (i18n/label :t/user-gets {:name receiver})
      :subtitle        amount}]])
 
+(defn select-asset-bottom-sheet
+  []
+  (let [{preselected-token-symbol :symbol} (rf/sub [:wallet/wallet-send-token])]
+    [:<> ;; Need to be a `:<>` to keep `asset-list` scrollable.
```

Comment:
Are you sure we need this comment?

Reply:
I don't think someone will try to remove this `:<>` and put a `:view` here, but it can work as a reference for future  scrolllable views inside a bottom sheet, I spent time realizing about it because it's a bit tricky, so I thought it was a good idea to add it.

do you think it should be removed?